packages/editor/src/store/effects/posts.js

```diff
@@ -120,8 +120,8 @@ export const requestPostUpdate = async ( action, store ) => {
 			data: toSend,
 		} );
 	} else {
-		dispatch( removeNotice( SAVE_POST_NOTICE_ID ) );
-		dispatch( removeNotice( AUTOSAVE_POST_NOTICE_ID ) );
+		dataDispatch( 'core/notices' ).removeNotice( SAVE_POST_NOTICE_ID );
+		dataDispatch( 'core/notices' ).removeNotice( 'autosave-exists' );
```

Comment:
Why do we stop using this `AUTOSAVE_POST_NOTICE_ID` constant? Are there any blockers? Can't we use the trick with `SAVE_POST_NOTICE_ID`?

Reply:
I kinda feel these constants are a waste of our resources, in the same way we don't bother with action type constants. I'll see about at least making things consistent, even if that means reintroducing the constant.

Reply:
> Why do we stop using this `AUTOSAVE_POST_NOTICE_ID` constant? Are there any blockers? Can't we use the trick with `SAVE_POST_NOTICE_ID`?

Looking at this more, I don't really like the idea of importing store constants into the `EditorProvider` component. And in general, I don't really like how we've implemented this notion of notices which should be dismissed on the next save. Rather than hard-coding their removal, it seems like it might be better to have some separate state tracking IDs of notices which should be removed on the next save. This way the `EditorProvider` could add to the state at the same time it's creating its notice, keeping the ID colocated.

Thoughts? Maybe better to defer to a follow-up pull request, keeping this one as-is?

Reply:
I’m on mobile so I can’t check the last part of your comment. It sounds like a good idea. We can discuss in the future. Let’s get this in first 😃